x-pack/plugins/timelines/public/components/t_grid/body/helpers.tsx

```diff
@@ -239,3 +239,12 @@ export const addBuildingBlockStyle = (
     });
   }
 };
+
+/** Returns true when the specified column has cell actions */
+export const hasCellActions = ({
+  columnId,
+  disabledCellActions,
```

Comment:
Suggestion: what about setting a default arg to `FIELDS_WITHOUT_CELL_ACTIONS` from `x-pack/plugins/security_solution/public/common/lib/cell_actions/constants.ts`?
```suggestion
  disabledCellActions = FIELDS_WITHOUT_CELL_ACTIONS,
```

Reply:
in this case it might be (slightly) safer to ensure consumers explicitly provide this input (even if it's an empty set), because we can't guarantee those fields would be applicable to every use case / app